sample_configs/cloud_configs.yaml

```diff
@@ -55,5 +55,5 @@ module_configs:
         - vehicle
     amlb_constraint:
       - test
-    amlb_custom_branch:
-      - https://github.com/gidler/autogluon#gidler_master_w_logging
+    # amlb_custom_branch:
+    #   - https://github.com/account/autogluon#custom_branch
```

Comment:
Should we put something to demonstrate to the reader that `account` and `custom_branch` need to be substituted. Maybe `<ACCOUNT>` and `<CUSTOM_BRANCH>` or something of that sort?

Reply:
Good point!